servers/pollux/default.nix

```diff
@@ -0,0 +1,163 @@
+{ modulesPath, inputs, config, pkgs, ... }:
+let
+  vs = config.vault-secrets.secrets;
+
+  suitecrm = import ./suitecrm.nix { inherit pkgs; system = pkgs.system; };
+in {
+  imports = [
+    inputs.serokell-nix.nixosModules.ec2
+    inputs.hermetic.nixosModules.hermetic
+  ];
+
+  networking.firewall.allowedTCPPorts = [ 80 443 ];
+
+  users.users.suitecrm = {
+    isNormalUser = true;
+    home = "/var/www/suitecrm";
+    uid = 1024;
+    group = "suitecrm";
+  };
+
+  users.groups.suitecrm = {
+    gid = 1024;
+  };
+
+  systemd.services.nginx.preStart = pkgs.lib.mkForce "";
+
+  systemd.services.suitecrm =
+    { before = [ "phpfpm-suitecrm.service" ];
+      wantedBy = [ "multiuser.target" ];
+      script =
+        ''
```

Comment:
Is this script copied from somewhere?

Reply:
nope, my own creation

Reply:
the idea here was to copy the php app itself into somewhere, where it can write into, but also move parts which are state into a separate folder. One dir I didn't move is `modules` as php relative imports are completely broken across symlinks. I'm also not really sure where else it might want to write that's why I gave it the freedom to write anywhere

Reply:
If it's the only way, then that's fine. I've settled on many similar solutions before, but if it's possible to safely avoid copying as many things into mutable directories then that's always preferable, these things always end up blowing up eventually.